action.yml

```diff
@@ -40,16 +40,17 @@ runs:
           branch=${GITHUB_REF#refs/heads/}
         fi
 
-        echo "##[set-output name=branch;]$(echo $branch)"
+        echo "branch_name=$branch" >> $GITHUB_OUTPUT
 
     - name: Extract release name
       id: extract_release
       shell: bash
       run: |
-        branch=${{ steps.extract_branch.outputs.branch }}
-        truncated_branch=$(echo $branch | tr '[:upper:]' '[:lower:]' | sed 's:^\w*\/::' | tr -s ' _/[]().' '-' | cut -c1-30 | sed 's/-$//')
+        branch_name=${{ steps.extract_branch.outputs.branch_name }}
+        truncated_branch=$(echo $branch_name | tr '[:upper:]' '[:lower:]' | sed 's:^\w*\/::' | tr -s ' _/[]().' '-' | cut -c1-30 | sed 's/-$//')
```

Comment:
I appreciate this hasn't changed, but I wonder if it's worth documenting how each piped operation mutates the branch name?

If this broke, how would we fix it? I know I'd spend ages trying to figure out what was going on first!